SPECS/freetds/freetds.spec

```diff
@@ -1,8 +1,8 @@
 %define bits	%{?__isa_bits:%{__isa_bits}}%{!?__isa_bits:32}
 Summary:        Implementation of the TDS (Tabular DataStream) protocol
 Name:           freetds
-Version:        1.1.20
-Release:        5%{?dist}
+Version:        1.3.20
```

Comment:
Curious if we can move forward to v1.4.10 which was released recently

Reply:
Ack. Sure :)

Reply:
Done